templates/news/news-en.html

```diff
@@ -443,8 +449,7 @@ <h2 class="wb-inv">Search results</h2>
 				</tr>
 				</thead>
 			</table>
+			<p class="text-center nojs-hide"><a href="https://www.canada.ca/en/news/advanced-news-search/news-results.html">Access all news products</a>.</p>
```

Comment:
Please remove the extra dot at the end of "Access all news products" since it's not part of the BR.